articles/azure/azs-how-config-vm-update-management.md

```diff
@@ -0,0 +1,203 @@
+
+    - Click **Create a resource**.
+
+    - In the search bar, search for `log analytics`.
+
+        ![Log Analytics search](images/azs-browser-search-log-analytics.png)
+
+    - Click **Log Analytics**.
+
+    - Click **Create**.
+
+    - Provide the following:
+
+        - A name for the **Log Analytics Workspace**. Example: `DefaultLAWorkspace`.
+
+        - A **Subscription** to link the workspace to.
+
+        - A **Resource group** to host the workspace in.
+
+        - A **Location** to host the workspace in.
+
+        > [!NOTE]
+        > The currently supported locations are: *West Central US*, *East US*, *Canada Central*, *UK South*, *West Europe* and *Southeast Asia*.
+
+        - A **Pricing tier** to use.
+
+        The following example shows the blade with all items filled out:
+
+        ![Log Analytics workspace creation](images/azs-browser-example-log-analytics-workspace.png)
+
+    - Click **OK** once finished. The Log Analytics workspace will now begin deploying. A notification will appear in the top right of the portal.
+
+    - Once deployment is complete, navigate to the resource group you placed the Log Analytics workspace in.
+
+    - Select the newly created workspace. On the new blade, under *Settings*, select **Advanced settings**, **Connected Sources** and then **Windows Servers** or **Linux Servers** depending on the VM type you want to add analytics to.
+
+    - Note down the **Workspace ID** and **Primary Key** values.
+
+       ![Log Analytics workspace advanced settings](images/azs-browser-log-analytics-workspace-advanced-settings.png)
+
+    - Within *Advanced settings*, select the *Data* blade and click **Windows Performance Counters** or **Linux Performance Counters**.
+
+    - Ensure all counters are selected, then click **Add the selected performance counters**.
+
+    - Click **Save**.
+
+        ![Performance counters](images/azs-browser-example-performance-counters.png)
+
+        > [!NOTE]
+        > For Linux performance counters, ensure **Apply below configuration to my machines** is selected.
+
+3. Create and configure an *Automation Account*:
+
+    - Click **Create a resource**.
+
+    - Search for `automation`.
+
+    - Select **Automation** and then click **Create**.
+
+        ![Automation Account](images/azs-browser-search-automation-account.png)
+
+    - Choose a **Name**, **Subscription**, **Resource group**, **Location** and ensure **Create Azure Run As account** is set to **Yes**.
+
+    - Click **Create**.
+
+        ![Add Automation Account](images/azs-browser-add-automation-account.png)
+
+        > [!NOTE]
+        > The error "Azure Classic Run As account creation error" may occur. This **DOES NOT** affect the process.
+
+4. Once deployed, navigate to the *Automation Account*. In the new blade, under *Configuration Management*, select the solution to enable (**Inventory & Change Tracking** or **Update Management**).
```

Comment:
Select the solution to enable (Inventory or Change tracking).

- Repeat step 4 for Update Management under the *Update Management*